internal/monitors/traceforwarder/forwarder.go

```diff
@@ -60,3 +63,9 @@ func setupHandler(ctx context.Context, router *mux.Router, chainType string, sin
 	pathSetup(router, handler, signalfx.DefaultTracePathV1)
 	return internalMetrics
 }
+
+func notFoundHandler(w http.ResponseWriter, r *http.Request) {
+	errMsg := fmt.Sprintf("Trace span request received on invalid path '%s'. You should send to the same path that you would on the Smart Gateway.", r.URL.Path)
+	logger.Error(errMsg)
```

Comment:
Is this a throttled logger? I can see quickly overloading the log output of the local host's agent if applications are misconfigured.

Reply:
Good point, let me see how to best handle that.